File: Phase11/Back end/Phase05/IO/Output.cs
using System;
using System.Collections.Generic;

namespace Phase05.IO
{
    public class Output
    {
        public static void PrintSet(HashSet<int> result)
        {
            foreach (int s in result)
            {
                Console.WriteLine(s);
            }
        }

        public static void PrintStringList(List<string> list)
        {
            for (int i = 0; i < list.Count; i++)
            {
                Console.WriteLine("-------Document" + i + "\n:" + list[i]);
            }
        }

        public static void PrintString(string str)
        {
            Console.WriteLine(str);
        }
    }
}

File: Phase01/Output.java
import java.util.ArrayList;
import java.util.HashSet;

public class Output
{
    public static void printSet(HashSet<Integer> set, ArrayList<String> docs)
    {
        for (Integer s: set)
        {
            System.out.println(s);
            System.out.println("------------------This is document:");
            System.out.println(docs.get(s));
            System.out.println();
            System.out.println();
            System.out.println();
            System.out.println();
            System.out.println();
        }
    }
}